diff --git a/ofproto/ofproto.c b/ofproto/ofproto.c
index 5688c6e..ca7d40a 100644
--- a/ofproto/ofproto.c
+++ b/ofproto/ofproto.c
@@ -365,8 +365,6 @@ ofproto_init(const struct shash *iface_hints)
     struct shash_node *node;
     size_t i;
 
-    ofproto_class_register(&ofproto_dpif_class);
-
     /* Make a local copy, since we don't own 'iface_hints' elements. */
     SHASH_FOR_EACH(node, iface_hints) {
         const struct iface_hint *orig_hint = node->data;
